### ISSUE TITLE
Release Twisted 8.1

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3196]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3196|
|Type|task|
|Created|2008-04-20 15:40:51Z|
|Branch|https://github.com/twisted/twisted/tree/release-8.1.x-3196-2|

We debated doing 8.0.2 and 8.1, and we decided to go with 8.1 because no stable OSes have 8.0 yet and we have no need for restricting the release to bug fixes.

<details><summary>Searchable metadata</summary>

```
trac-id__3196 3196
type__task task
reporter__radix radix
priority__normal normal
milestone__ 
branch__branches_release_8_1_x_3196_2 branches/release-8.1.x-3196-2
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1208706051000000 1208706051000000
changetime__1211132397000000 1211132397000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix commented|
|-|-|

(In [23346]) Branching to 'release-8.1.x-3196'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix commented|
|-|-|

(In [23439]) Branching to 'release-8.1.x-3196-2'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix removed owner|
|-|-|

Please review.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve commented|
|-|-|

So for me the only remaining thing for me is remove all the pre1/pre2 mentions.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set owner to @radix|
|-|-|

Maybe put it back once pre1/pre2 have been removed.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_6"></a>|@radix commented|
|-|-|

up for review

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to @exarkun|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set owner to @radix|
|-|-|

Looks good.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_9"></a>|@radix set status to `closed`|
|-|-|

(In [23663]) Merge release-8.1.x-3196-2

Author: radix
Reviewer: exarkun
Fixes: [#3196](https://github.com/twisted/twisted/issues/3196)

Version changes and NEWS updates from the 8.1 release.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
